Failing unit test test_simple

With the current master, test_simple fails for me:

```..F
======================================================================
FAIL: test_simple (__main__.TestNinjaTracing)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./ninjatracing_test", line 41, in test_simple
    self.assertEqual(expected, dicts)
AssertionError: Lists differ: [{'args': {}, 'name': 'my_firs... != [{'args': {}, 'name': 'my_outp...

First differing element 0:
{'args': {}, 'name': 'my_first_output', 'tid': '0', 'ph': 'X', 'dur': '70000', 'pid': '42', 'ts': '50000', 'cat': 'targets'}
{'args': {}, 'name': 'my_output', 'tid': '0', 'ph': 'X', 'dur': '100000', 'pid': '42', 'ts': '100000', 'cat': 'targets'}

  [{'args': {},
+   'cat': 'targets',
+   'dur': '100000',
+   'name': 'my_output',
+   'ph': 'X',
+   'pid': '42',
+   'tid': '0',
+   'ts': '100000'},
+  {'args': {},
    'cat': 'targets',
    'dur': '70000',
    'name': 'my_first_output',
    'ph': 'X',
    'pid': '42',
-   'tid': '0',
-   'ts': '50000'},
-  {'args': {},
-   'cat': 'targets',
-   'dur': '100000',
-   'name': 'my_output',
-   'ph': 'X',
-   'pid': '42',
    'tid': '1',
-   'ts': '100000'}]
?          ^^

+   'ts': '50000'}]
?          ^


----------------------------------------------------------------------
Ran 3 tests in 0.002s

FAILED (failures=1)
```

--- a/ninjatracing
+++ b/ninjatracing
@@ -50,9 +50,8 @@ def read_targets(log, show_all):
             # so that this new build will be displayed independently.
             targets = {}
         last_end_seen = int(end)
-        start, end = -int(end), -int(start)
         targets.setdefault(cmdhash, Target(start, end)).targets.append(name)
-    return sorted(targets.values(), key=lambda job: job.start, reverse=True)
+    return sorted(targets.values(), key=lambda job: job.end, reverse=True)
 
 
 class Threads:
@@ -63,10 +62,10 @@ class Threads:
     def alloc(self, target):
         """Places target in an available thread, or adds a new thread."""
         for worker in range(len(self.workers)):
-            if self.workers[worker] <= target.start:
-                self.workers[worker] = target.end
+            if self.workers[worker] >= target.end:
+                self.workers[worker] = target.start
                 return worker
-        self.workers.append(target.end)
+        self.workers.append(target.start)
         return len(self.workers) - 1
 
 

--- a/ninjatracing
+++ b/ninjatracing
@@ -76,7 +76,7 @@ def log_to_dicts(log, pid, show_all):
     for target in read_targets(log, show_all):
         yield {
             'name': '%0s' % ', '.join(target.targets), 'cat': 'targets',
-            'ph': 'X', 'ts': str(-target.end * 1000),
+            'ph': 'X', 'ts': str(target.start * 1000),
             'dur': str((target.end - target.start) * 1000),
             'pid': str(pid), 'tid': str(threads.alloc(target)), 'args': {},
             }

--- a/ninjatracing_test
+++ b/ninjatracing_test
@@ -29,12 +29,12 @@ class TestNinjaTracing(unittest.TestCase):
                        "50\t120\t0\tmy_first_output\t0afef\n")
         dicts = list(ninjatracing.log_to_dicts(log, 42, True))
         expected = [{
-                'name': 'my_first_output', 'cat': 'targets', 'ph': 'X',
-                'ts': '50000', 'dur': '70000', 'pid': '42', 'tid': '0',
+                'name': 'my_output', 'cat': 'targets', 'ph': 'X',
+                'ts': '100000', 'dur': '100000', 'pid': '42', 'tid': '0',
                 'args': {}
                 }, {
-                'name': 'my_output', 'cat': 'targets', 'ph': 'X',
-                'ts': '100000', 'dur': '100000', 'pid': '42', 'tid': '1',
+                'name': 'my_first_output', 'cat': 'targets', 'ph': 'X',
+                'ts': '50000', 'dur': '70000', 'pid': '42', 'tid': '1',
                 'args': {}
                 },
             ]

